internal/distro/rhel84/distro_v2.go

```diff
@@ -532,6 +534,27 @@ func (t *imageTypeS2) userStageOptions(users []blueprint.UserCustomization) (*os
 	return &options, nil
 }
 
+func (t *imageTypeS2) usersFirstBootOptions(usersStageOptions *osbuild.UsersStageOptions) *osbuild.FirstBootStageOptions {
+	cmds := make([]string, 0, 2*len(usersStageOptions.Users))
```

Comment:
`3*`? `3*len()+1`? :)

Reply:
Good catch.  It's not biggie since it's just capacity, but might as well fix it.